docs/en/advanced/Command-Line-Interface.md

```diff
@@ -29,6 +11,253 @@ priority: 0
+* `--level <arg>` If provided the command changes to the defined logger level, 
+otherwise it gets and displays the current logger level.
+
+For example, this command sets the class `alluxio.heartbeat.HeartbeatContext`'s logger level to DEBUG on master as well as a worker at `192.168.100.100:30000`.
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=master,192.168.100.100:30000 --level=DEBUG
+```
+
+And the following command gets all workers' log level on class `alluxio.heartbeat.HeartbeatContext`
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
+```
+
+### runTests
+
+The `runTests` command will run all end-to-end tests on an Alluxio cluster 
+and provide a more comprehensive sanity check.
+
+```bash
+$ ./bin/alluxio runTests
+```
+
+### runJournalCrashTest
+
+The `runJournalCrashTest` command will test the Master Journal System in a crash scenario.
+
+Options:
+
+* `-creates <arg>` Number of Client Threads to request create operations
+* `-deletes <arg>` Number of Client Threads to request create/delete operations
+* `-maxAlive <arg>` The maximum time a master should ever be alive during the test, in seconds
+* `-renames <arg>` Number of Client Threads to request create/rename operations
+* `-testDir <arg>` Test Directory on Alluxio
+* `-totalTime <arg>` The total time to run this test, in seconds. This
+value should be greater than [maxAlive]
+
+```bash
+$ ./bin/alluxio runJournalCrashTest
+
+# Launches total 6 clients connecting to the Master and the Master will
+# crash randomly with the max alive time 5 seconds.
+$ ./bin/alluxio runJournalCrashTest -maxAlive 5 -totalTime 20 -creates 2 -deletes 2 -renames 2
+```
+
+### runMesosTest
+
+The `runMesosTest` command will test the Alluxio integration with Mesos.
+
+```bash
+$ ./bin/alluxio runMesosTest
+```
+
+### readJournal
+
+The `readJournal` command will read an Alluxio journal file from stdin 
+and write a human-readable version of it to stdout.
+
+Options: 
+
+* `-journalFile <arg>` If set, only read journal from this file. `-master` is ignored when `-journalFile` is set.
+* `-master <arg>` The name of the master (e.g. FileSystemMaster, BlockMaster). Set to FileSystemMaster by default.
+* `-start <arg>` The start log sequence number (inclusive). Set to 0 by default.
+* `-end <arg>` The end log sequence number (exclusive). Set to +inf by default.
+
+```bash
+$ ./bin/alluxio readJournal
+$ ./bin/alluxio readJournal -master BlockMaster 
+```
+
+### upgradeJournal
+
+The `upgradeJournal` command will upgrade an Alluxio journal version 0 (Alluxio version < 1.5.0) 
+to an Alluxio journal version 1 (Alluxio version >= 1.5.0).
+
+`-journalDirectoryV0 <arg>` will provide the v0 journal persisted location. It is assumed to be 
+the same as the v1 journal directory if not set.
+
+```bash
+$ ./bin/alluxio upgradeJournal
+```
+
+### copyDir
+
+The `copyDir` command will copy the `PATH` to all worker nodes.
```

Comment:
it will copy the path to all workers listed in `conf/workers`

Reply:
Added